### PR TITLE
Feature: Added systray icon of Files shown when running on the background

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

Users doesn't have a proper way to kill the app when running on the background. Added system tray icon to handle killing app by the app itself.

## Task Checklist

- [x] Added utility classes for the Taskbar
- [ ] Added functionalities to kill/resume

## Steps To Validate Changes

1. Launch the app.
2. Enable background process running.
3. Close window.
4. See notification area on the taskbar.

## Known Issues

_N/A_

## PR Checklist

- [x] **Development**: #14240
- [x] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [x] **Localization**: Modified en-US string resources[^2]
- [ ] **Co-authors**: _N/A_

## Screenshots

image

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -3659,4 +3659,10 @@
   <data name="SortPriority" xml:space="preserve">
     <value>Sort priority</value>
   </data>
+  <data name="Quit" xml:space="preserve">
+    <value>Quit</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:3663]:
```suggestion
    <value>Close Files</value>
```

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -3659,6 +3659,12 @@
   <data name="SortPriority" xml:space="preserve">
     <value>Sort priority</value>
   </data>
+  <data name="Quit" xml:space="preserve">
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:3662]:
```suggestion
  <data name="CloseFiles" xml:space="preserve">
```

Comment (0x5bfa) [reply src/Files.App/Strings/en-US/Resources.resw:3662]: I mistakenly merged your suggestion. IIRC we've decided what to call [before](https://discord.com/channels/725513575971684472/1164222926749442059/1164224058670448753). But I think we can decide again lol.

Comment (yaira2) [reply src/Files.App/Strings/en-US/Resources.resw:3662]: Either one works

Comment (0x5bfa) [reply src/Files.App/Strings/en-US/Resources.resw:3662]: User would be confused why have to close again. Idk but I often see quit but never close.

**Pull request closed without merging**